pkg/operator/controller/ingress/status_test.go

```diff
@@ -514,7 +518,7 @@ func Test_computeDeploymentRollingOutCondition(t *testing.T) {
 			expectMessageContains: "Deployment is not actively rolling out",
 		},
 		{
-			name:                  "Router pod replicas have/updated < want",
+			name:                  "Router pod replicas have/updated < wantGrace",
```

Comment:
It seems like from the code under test that this should be named `wantReplicas` or `replicasWanted` to avoid confusion with something like a grace period (time) which I don't think is being tested?